docs/processes/cldr-icu.md

```diff
@@ -166,9 +166,10 @@ export CLDR_DATA_DIR=$HOME/cldr-staging/production
 
 1c. ICU variables
 ```sh
-export ICU4C_DIR=$HOME/icu-myfork/icu4c
-export ICU4J_ROOT=$HOME/icu-myfork/icu4j
-export TOOLS_ROOT=$HOME/icu-myfork/tools
+export ICU_DIR=$HOME/icu-myfork
+export ICU4C_DIR=$ICU_DIR/icu4c
+export ICU4J_ROOT=$ICU_DIR/icu4j
+export TOOLS_ROOT=$ICU_DIR/tools
```

Comment:
During my testing of these changes, I tried to follow these instructions. I had to fix the documentation so that I could test these changes. These instructions were tested on macOS.